.docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11-slim
+LABEL maintainer="adamj"
+
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE 1 \
+    PYTHONUNBUFFERED 1
+
+RUN apt-get update \
```

Comment:
(blocking; optimizing)
Is such number of RUN commands optimal for Docker's layers structure?